GPSTest/src/main/java/com/chesapeaketechnology/gnssmonkey/AGpsMonkeyActivity.java

```diff
@@ -66,16 +69,18 @@ protected void onCreate(Bundle savedInstanceState) {
 //        openBatteryOptimizationDialogIfNeeded();
     }
 
+    /**
+     * Starts the GPS Monkey service which handles logging the GNSS data to a GeoPackage file.
+     */
     private void startService() {
         // TODO KMB: I don't think this case can happen. Even when Android was forced to kill the
```

Comment:
And then initialize it here:
```suggestion
        if (serviceIntent == null) serviceIntent = new Intent(this, GpsMonkeyService.class);
        
        // TODO KMB: I don't think this case can happen. Even when Android was forced to kill the
```